tests/cypress/e2e/form/editor/conditions.cy.js

```diff
@@ -157,7 +157,7 @@ function initVisibilityConfiguration() {
 }
 
 function closeVisibilityConfiguration() {
-    cy.get('body').type('{esc}');
+    cy.get('[data-glpi-form-editor-visibility-dropdown]:visible').click();
```

Comment:
Pressing 'esc' while a date input is focused throw some error in cypress so I had to use another solution for this.